app/models/db/Metrics.scala

```diff
@@ -128,3 +113,16 @@ case class Fork(
     time: DateTime,
     wordCount: Int,
     revisionNumber: Int)
+
+object Fork {
+  def apply(forkData: ForkData): Fork =
+    new Fork(id = UUID.randomUUID.toString, forkData.composerId, forkData.time, forkData.wordCount, forkData.revisionNumber)
+
+  def customApply(tuple: (String, String, DateTime, Int, Int)): Fork =
+    Fork(
+      id = tuple._1,
```

Comment:
In general, I'm not a fan of `_1` style notation, but I think it's clear in this context.

Reply:
Not a fan either, but because I'm using a companion object I've lost the `tupled` method. I can't find a nice work around it.

Reply:
In this context, I think it's fine.